src/Symfony/Component/Clock/MockClock.php

```diff
@@ -20,13 +20,18 @@
  */
 final class MockClock implements ClockInterface
 {
-    private DatePoint $now;
+    private ?DatePoint $now = null;
+    /**
+     * @var (\Closure(\DateTimeZone $timezone): DatePoint)|null
```

Comment:
please also add the `@param` on the constructor to document the signature of the closure supported in the argument. This is a lot more useful than on the private property (the parameter documentation impacts the public API and so will help when running static analysis in projects using the MockClock, while the private property impacts only the Symfony codebase)